cumulus/polkadot-omni-node/lib/src/cli.rs

```diff
@@ -128,7 +128,8 @@ pub struct Cli<Config: CliConfig> {
 	///
 	/// This is a dev option, and it won't result in starting or connecting to a parachain network.
 	/// The resulting node will work on its own, running the wasm blob and artificially producing
-	/// a block each `dev_block_time` ms, as if it was part of a parachain.
+	/// a block each `dev_block_time` ms, as if it was part of a parachain. Defaults to 3000ms if
+	/// not set and `--dev` is used.
```

Comment:
I would be more explicit here about enabling manual sealing and fact that `--dev` enables this.
```
/// Start a dev node that produces a block each `dev_block_time` ms.
///
/// This is a dev option. It enables a manual sealing, meaning blocks are produced manually 
/// rather than being part of an actual network consensus process. Using the option won't 
/// result in starting or connecting to a parachain network. The resulting node will work on
/// its own, running the wasm blob and artificially producing a block each `dev_block_time` ms, 
/// as if it was part of a parachain. 
///
/// The `--dev` flag sets the `dev_block_time` to a default value of 3000ms unless explicitly provided.
```